velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -136,7 +136,7 @@ class SumAggregate
   /// For floating points, an overflow results in +/- infinity which is a
   /// valid output.
   template <typename TOutput>
-  static void updateSingleValue(TOutput& result, TInput value) {
+  static void updateSingleValue(TOutput& result, TOutput value) {
```

Comment:
Shouldn't here be TInput? Thanks!

Reply:
The comment  https://github.com/facebookincubator/velox/pull/2225#issuecomment-1208850511 explains why
`updateNonNullValue` requires a change.
That call invokes `updateValue` template which can either be `updateSingleValue` or `updateDuplicateValues`.
So these have to change as well.

Reply:
I see. Thanks for the explanation! Using TOutput is a little bit confusing here. How about using TData as the template type name? Thanks!

Reply:
Sounds good. Done.